gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapProvider.java

```diff
@@ -73,29 +74,28 @@ public Fragment getFragment() {
 
   @Override
   public int getMapType() {
-    if (map != null) {
-      return map.getValue().getMapType();
+    if (map == null) {
+      throw new IllegalStateException("MapAdapter is null");
     }
-    throw new IllegalStateException("MapAdapter is null");
+    return map.getValue().getMapType();
   }
 
   @Override
   public void setMapType(int mapType) {
-    if (map != null) {
-      map.getValue().setMapType(mapType);
-    } else {
+    if (map == null) {
       throw new IllegalStateException("MapAdapter is null");
     }
+    map.getValue().setMapType(mapType);
   }
 
   @Override
-  public HashMap<Integer, String> getMapTypes() {
-    HashMap<Integer, String> hashMap = new HashMap<>();
-    hashMap.put(GoogleMap.MAP_TYPE_NONE, "None");
-    hashMap.put(GoogleMap.MAP_TYPE_NORMAL, "Normal");
-    hashMap.put(GoogleMap.MAP_TYPE_SATELLITE, "Satellite");
-    hashMap.put(GoogleMap.MAP_TYPE_TERRAIN, "Terrain");
-    hashMap.put(GoogleMap.MAP_TYPE_HYBRID, "Hybrid");
-    return hashMap;
+  public Map<Integer, String> getMapTypes() {
```

Comment:
Please use an ImmutableMap<> defined in a constant instead. The constant can then be returned from this method.

Reply:
Also add `// TODO: i18n.`